### ISSUE TITLE
Update for tf2.0

### ISSUE THREAD

**Issue by bmcfee** [bmcfee/crema#28]
crema currently breaks horribly on tensorflow 2.0.  Probably we will need to update things in pumpp as well.

**Comment by kylemcdonald** [comment]
fwiw i managed to run successfully with tf2.1, but when i tried to go back to tf1.15 i got an error:

```
InternalError: 2 root error(s) found.
  (0) Internal: cuDNN launch failure : input shape ([1,2,1616,216])
	 [[{{node batch_normalization_1/cond/FusedBatchNormV3}}]]
	 [[chord_root/Reshape_1/_277]]
  (1) Internal: cuDNN launch failure : input shape ([1,2,1616,216])
	 [[{{node batch_normalization_1/cond/FusedBatchNormV3}}]]
0 successful operations.
0 derived errors ignored.
```

i didn't test any other tf1.15 code, so this might just be CUDA and not related to this repo. i have 10.1 installed and i'm not sure 1.15 is compatible.

**Comment by bmcfee** [comment]
fixed by merging #36
